assets/queries/k8s/allowed_capabilities_psp/test/positive_expected_result.json

```diff
@@ -0,0 +1,7 @@
+[
+	{
+		"queryName": "Allowed Capabilities PSP",
```

Comment:
Change the query's name to be different. Maybe something like "PSP Has Allowed Capabilities".